app/models/manageiq/providers/redhat/infra_manager/event_parser.rb

```diff
@@ -1,3 +1,5 @@
+require 'ostruct'
```

Comment:
Can you move this require down to where you're using OpenStruct? https://github.com/ManageIQ/manageiq-providers-ovirt/pull/467/files#diff-52fa558b219d05b5565f5fc8ea42e88cR214

We try to keep global requires down so code that doesn't need this doesn't have to load it for memory usage reasons

Reply:
Yes sure